### PR TITLE
UI/fix leftovers

### PR DESCRIPTION
**Pull request by odedre**
## Proposed Changes

- use box - pass version if not latest
- added tooltip to menu widgets and fixed icon sizes
- compositions panel styles
- change log fix for avatars
- fix for drawer scroll styles

### CODE REVIEW

File: scopes/compositions/compositions/ui/compositions-panel/compositions-panel.module.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
   font-size: var(--p-xs);
   &:hover {
     background-color: var(--border-color, #ededed);
```
Comment (KutnerUri) [inline scopes/compositions/compositions/ui/compositions-panel/compositions-panel.module.scss:9]: nice

File: scopes/ui-foundation/menu-widget-icon/menu-widget-icon.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import ReactTooltip from 'react-tooltip';
+import styles from './menu-widget-icon.module.scss';
+
+type MenuWidgetIconProps = {
+  tooltipContent: ReactNode;
+  id: string;
+  icon: string;
+} & HTMLAttributes<HTMLDivElement>;
+
+export function MenuWidgetIcon({ tooltipContent, id, icon }: MenuWidgetIconProps) {
```
Comment (KutnerUri) [inline scopes/ui-foundation/menu-widget-icon/menu-widget-icon.tsx:12]: cool, consider renaming to MenuWidget

File: scopes/component/component/ui/menu/menu.tsx [hunk 3]
```diff
@@ -32,13 +36,20 @@ export function Menu({ navigationSlot, widgetSlot, className, host, menuItemSlot
   const { component } = useComponent(host);
   const mainMenuItems = useMemo(() => groupBy(flatten(menuItemSlot.values()), 'category'), [menuItemSlot]);
 
-  if (!component) return <FullLoader />;
+  const versionList =
+    useMemo(
+      () =>
+        component?.tags
+          ?.toArray()
+          .map((tag) => tag?.version?.version)
```
Comment (KutnerUri) [inline scopes/component/component/ui/menu/menu.tsx:44]: filter undefineds?

File: scopes/ui-foundation/uis/main-dropdown/main-dropdown.module.scss [hunk 4]
```diff
@@ -34,17 +37,8 @@
     background-color: var(--bg-dent);
   }
 }
-
-// TODO  @oded - @hack! this is for closing the menu when clicking an iframe for the beta. we need a better way to do this because it uses a global data-attr
```
Comment (KutnerUri) [inline scopes/ui-foundation/uis/main-dropdown/main-dropdown.module.scss:38]: <3

**Review by KutnerUri — APPROVED** [review]
nice work man!

File: scopes/component/component/ui/menu/menu.tsx (hunk 3 above)
Comment (odedre) [reply scopes/component/component/ui/menu/menu.tsx:44]: I moved the if further down to be able to use `useMemo` so component might be undefined here

Comment (odedre) [reply scopes/component/component/ui/menu/menu.tsx:44]: but I can also remove the use memo from the version list or extract to function so I will still have the `if(!component)` before the versions

Comment (odedre) [reply scopes/component/component/ui/menu/menu.tsx]: fixed

**Pull request merged**